Updates to the Wiki for UM39 and Beyond (#15)
Ultramarine-Linux/wiki-new: src/content/docs/en/development/ux-guide.mdx

import Alert from "../../../../components/Docs/Alert.astro";

### The Helium Human Interface Guidelines Can Be Found on [Fyra Developer](https://developer.fyralabs.com/helium/hig) 

> We now recommend the Helium Human Interface Guidelines. They're built in house to provide a modern and friendly experience. Although they are designed for the Helium UI Library, you can apply the principles to any UI library.

Reviewer: ```suggestion
We recommend the Helium Human Interface Guidelines. They're built in house to provide a modern and friendly experience. They were originally designed for the Helium UI Library but you can apply the principles to any UI libraries nonetheless.
```
